This file is part of MXE. See LICENSE.md for licensing information.

Contains a fix for building on QT 5.13.

From 1b4f22431ae35dfd11f07a5ae88a1b4db3de2a85 Mon Sep 17 00:00:00 2001
From: Michael Pyne <mpyne@kde.org>
Date: Tue, 18 Dec 2018 17:47:21 -0500
Subject: [PATCH] Fix build with Qt 5.13 / GCC 8.2.

At this point Grantlee doesn't build for me because QList is an
incomplete type. I think the compiler is right to complain, so I fix by
including the appropriate header.

Fixes issue #47.
---
 templates/lib/lexer_p.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/lib/lexer_p.h b/templates/lib/lexer_p.h
index 275aeea..1ae451a 100644
--- a/templates/lib/lexer_p.h
+++ b/templates/lib/lexer_p.h
@@ -24,7 +24,7 @@
 #include "textprocessingmachine_p.h"
 #include "token.h"
 
-template <typename T> class QList;
+#include <QList>
 
 namespace Grantlee
 {
